doc/contributor/cexts.md

```diff
@@ -1,5 +1,10 @@
 # TruffleRuby, Sulong and C extensions
 
+## Requirements
+
+* `llvm` including the `opt` binary
+* OpenSSL 1.0.2 including the development headers
```

Comment:
It's linked just below, should we add the link a second time?
Or add a sentence "See below on how to install these requirements." ?